src/main/apiClient.ts

```diff
@@ -183,46 +183,53 @@ export default class ApiClient {
   }
 
   loginFromSession(opts: AuthOptions = {}): Promise<void> {
-    if (!this.config.sso && !opts.idTokenHint) {
+    if (!this.config.sso && !opts.idTokenHint)
       return Promise.reject(
         new Error("Cannot call 'loginFromSession' without 'idTokenHint' parameter if SSO is not enabled.")
       )
-    }
 
     return this.loginWithRedirect({
       ...this.authParams({
         ...opts,
-        useWebMessage: false
+        useWebMessage: false,
       }),
       prompt: 'none'
     })
   }
 
   checkSession(opts: AuthOptions = {}): Promise<AuthResult> {
-    if (!this.config.sso && !opts.idTokenHint) {
+    if (!this.config.sso && !opts.idTokenHint)
```

Comment:
```suggestion
    if (this.config.sso)
```
Same as above. We need to remove this asap as it's been fixed at the backend a while ago.